allcontainers.js

```diff
@@ -62,6 +63,12 @@ function allContainers (opts) {
     }
 
     switch (data.status) {
+      case 'unpause':
+        if (pausedPreheatContainers[data.id]) {
+          pausedPreheatContainers[data.id] = null;
```

Comment:
delete here is actually the way to go, @mcdonnelldean :).

You don't want to maintain old stuff around. Even better, we could use an ES6 Map (which will be better).